test/cpp/jit/test_cleanup_passes.cpp

```diff
@@ -4,6 +4,7 @@
 #include <torch/csrc/jit/ir/ir.h>
 #include <torch/csrc/jit/ir/irparser.h>
 #include <torch/csrc/jit/testing/file_check.h>
+#include <torch/csrc/jit/frontend/run_cleanup_passes.cpp>
```

Comment:
Why is a cpp file being included here?

Reply:
See: https://github.com/pytorch/pytorch/pull/59245#discussion_r644241674

Praying Nikita will save me, but would also welcome suggestions

Reply:
What's the linker error you are seeing?

Reply:
I caught a glimpse (~2s) when discussing it with @ansley. I think the error complains about some symbol visibility. Maybe it is caused by cross-library (libtorch vs libtorch_python) access of public symbols. In that case, one common solution is to add `TORCH_API` modifier in front of declaration of that symbol, which modifies the visibility. 

If it doesn't help, please post the detailed error here so that @SplitInfinity or I can help take a look.

Reply:
Awesome, adding `TORCH_API` fixed it